go.mod

```diff
@@ -77,13 +78,13 @@ require (
 	gomodules.xyz/jsonpatch/v2 v2.0.1
 	gopkg.in/mgo.v2 v2.0.0-20180705113604-9856a29383ce // indirect
 	gopkg.in/yaml.v2 v2.2.4
-	k8s.io/api v0.0.0
+	k8s.io/api v0.15.8
```

Comment:
changing `k8s.io/*` version here is not quite helpful, because there are plenty of `replace` directives below to change the specific version.
the tricks here are related to depending on `k8s.io/kubernetes` directly, and we need to use `hack/pin-deps.sh` to pin to the specific kubernetes version

Reply:
fixed